example/elm.json

```diff
@@ -16,10 +14,9 @@
             "elm/http": "1.0.0",
             "elm/json": "1.0.0",
             "elm/time": "1.0.0",
-            "rtfeldman/elm-css": "16.0.0"
+            "rtfeldman/elm-css": "16.1.0"
         },
         "indirect": {
-            "Skinney/murmur3": "2.0.8",
```

Comment:
... I don't know where this was an indirect dep, but
1. it's been renamed so this fails
2. removing it seems to still work so 🤷‍♀️